src/components/pages/requests/CommentDisplay.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { sortArrayDesd as sortComments } from 'tesla-error-handler';
+import { EditorState, convertToRaw, ContentState } from 'draft-js';
+import { Editor } from 'react-draft-wysiwyg';
+import draftToHtml from 'draftjs-to-html';
```

Comment:
What are you using these for?
If they are not used in your component, I think you can remove them.